doc/source/whatsnew/v0.24.0.rst

```diff
@@ -1515,6 +1515,7 @@ MultiIndex
 I/O
 ^^^
 
+- Fix ``ValueError`` when reading a Dataframe with HDFStore in Python 3 from fixed format written in Python 2 (:issue:`24510`)
```

Comment:
can you move below (e.g. add it right before Plotting)

Reply:
Sorry but I didn't understand your request, I'm not familiar with rst format

Reply:
Ok, I got It, I moved my entry in the file